po/fr.po

```diff
@@ -5722,43 +5732,41 @@ msgstr ""
 
 #: /tmp/fish/implicit/share/completions/aptitude.fish:4
 msgid "Display a brief help message. Identical to the help action"
-msgstr ""
+msgstr "Afficher un bref message d’aide, Identique à l’action help"
 
```

Comment:
Sub clause, hence needs “identique” to be lowercase.

Reply:
Shouldn't this be to sentences as in the original text?

Reply:
I changed it as two sentences seemed useless because they're about the same thing, and it seems more natural in French to merge them with a comma separating the clauses.